test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageSourceFilesTest.php

```diff
@@ -191,31 +193,30 @@ public function provideExcludePattern(): array
                 [
                     'ClassA.php',
                     'tests/ATest.php',
-                    'foo/Tests/BTest.php',
+                    'foo/Bar/BTest.php',
                     'foo/src/ClassB.php',
-                    'foo/src/Tests/CTest.php',
-
+                    'foo/src/Bar/CTest.php',
                 ],
             ],
             'Exclude single directory by pattern' => [
                 ['/tests/'],
                 [
                     'ClassA.php',
-                    'foo/Tests/BTest.php',
+                    'foo/Bar/BTest.php',
                     'foo/src/ClassB.php',
-                    'foo/src/Tests/CTest.php',
+                    'foo/src/Bar/CTest.php',
                 ],
             ],
             'Exclude all subdirectories by pattern' => [
-                ['**/Tests/'],
+                ['**/Bar/'],
                 [
                     'ClassA.php',
                     'tests/ATest.php',
                     'foo/src/ClassB.php',
                 ],
             ],
             'Combine multiple patterns' => [
-                ['/tests/', '**/Tests/'],
+                ['/tests/', '**/Bar/'],
```

Comment:
Can some of these changes be reverted? It is kinda hard to understand whether they are due to changes in the code, or just refactoring of tests: I'd rather only see test additions

Reply:
Since there is already a "tests" pattern (with a lowercase "t"), I'd rather avoid potential case-insensitive issues by using a completely different word.